lesson5.js

```diff
@@ -0,0 +1,30 @@
+  const ul = document.getElementById('js-list');
+  const info = [
+    {to: "bookmark.html", img: "1.png", alt:"画像1", text: "ブックマーク"},
+    {to: "message.html", img: "2.png", alt:"画像2", text: "メッセージ"}
+    ];
+  const fragment = document.createDocumentFragment();
+
+  function list() {
```

Comment:
[ IMO ]
If the function name is ` list()`  I thought it would be difficult to understand what kind of processing it does.
I usually try to include verbs

If you would like, please check [this](https://dmitripavlutin.com/coding-like-shakespeare-practical-function-naming-conventions/#2-prefer-explanatory-names) article.

Reply:
Thank you for your advice that adding a verb will make it easier to understand. I learned that it is important to make it easier for others to understand what they are doing. I want to be careful when giving a function name.